agimus_demo_05_pick_and_place/agimus_demo_05_pick_and_place/urdf/franka_hand.xacro

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="utf-8"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="hand">
+  <!-- safety_distance: Minimum safety distance in [m] by which the collision volumes are expanded and which is enforced during robot motions -->
+  <xacro:macro name="franka_hand" params="connected_to:='' arm_id rpy:='0 0 0' xyz:='0 0 0' tcp_xyz:='0 0 0.1034' tcp_rpy:='0 0 0' safety_distance:=0 gazebo:=false description_pkg:=franka_description">
+    <xacro:unless value="${connected_to == ''}">
+      <joint name="${arm_id}_hand_joint" type="fixed">
+        <parent link="${connected_to}" />
+        <child link="${arm_id}_hand" />
+        <origin xyz="${xyz}" rpy="${rpy}" />
+      </joint>
+    </xacro:unless>
+
+    <xacro:link_with_sc name="hand" gazebo="${gazebo}">
+      <self_collision_geometries>
+      </self_collision_geometries>
+    </xacro:link_with_sc>
+
+    <!-- Define the hand_tcp frame -->
+    <link name="${arm_id}_hand_tcp" />
+    <joint name="${arm_id}_hand_tcp_joint" type="fixed">
+      <origin xyz="${tcp_xyz}" rpy="${tcp_rpy}" />
+      <parent link="${arm_id}_hand" />
+      <child link="${arm_id}_hand_tcp" />
+    </joint>
+    <link name="${arm_id}_leftfinger">
+      <visual>
+        <geometry>
+          <mesh filename="package://${description_pkg}/meshes/visual/finger.dae" />
```

Comment:
This mesh does not exists on `franka_description`. Is it something like
```suggestion
          <mesh filename="package://${description_pkg}/meshes/robot_ee/franka_hand_white/visual/finger.dae" />
```

Reply:
indeed, I wonder why it runs for me...